packages/perseus/src/widgets/radio/choice-icon.jsx

```diff
@@ -131,34 +138,43 @@ class LibraryChoiceIcon extends React.Component<{
             checked,
             showCorrectness,
             pressed,
+            multipleSelect,
             primaryProductColor,
             correct,
             transparentBackground,
         } = this.props;
+
+        let backgroundColor;
+        let borderColor;
+        let color;
         if (!showCorrectness && pressed) {
-            return {
-                borderColor: primaryProductColor,
-                color: primaryProductColor,
-                backgroundColor: transparentBackground
-                    ? "transparent"
-                    : styleConstants.white,
-            };
-        }
-        if (checked) {
+            borderColor = primaryProductColor;
+            color = primaryProductColor;
+            backgroundColor = transparentBackground
+                ? "transparent"
+                : styleConstants.white;
+        } else if (checked) {
             // Note: kaGreen is not only the default product color,
             // but also the "correctness" color
             const bg =
                 showCorrectness && correct ? Color.green : primaryProductColor;
-            return {
-                color: styleConstants.white,
-                backgroundColor: bg,
-                borderColor: bg,
-            };
+            color = styleConstants.white;
+            backgroundColor = bg;
+            borderColor = bg;
+        } else {
+            borderColor = Color.offBlack64;
+            color = Color.offBlack64;
         }
-        return {
-            borderColor: Color.offBlack64,
-            color: Color.offBlack64,
-        };
+
+        // define shape
+        let borderRadius;
+        if (multipleSelect) {
+            borderRadius = 3;
+        } else {
+            borderRadius = LIBRARY_ICON_SIZE;
+        }
```

Comment:
**style nit:** If this had been moved above `if (!showCorrectness && pressed) {`, then the changes to this function could've been minimized, but not having to introduce a bunch of `let` variables above.

I see that `constructStyles` does the same thing so making the code more consistent is a good thing, although I have a preference for `const` over `let`.

No action req'd.

Reply:
I'm not 100% sure I follow, but I tried to clean it up. I tend to favor `const` too, but with semi-confusing logic like this I favor readability. I tried to find a balance though, hopefully this works.